### PR TITLE
Added Wallet class

### PR DESCRIPTION
**Pull request by Dru27**
Wallet class with methods: winBattle, loseBattle and getMoneyInWallet. Unit tests are also included.

### CODE REVIEW

**Review by eliosfg — CHANGES_REQUESTED** [review]
it's ok, please review my feedback

File: src/main/java/org/fundacionjala/at15/pokemon/Wallet.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    public int winBattle() {
+        return moneyInWallet + BONUS;
+    }
+
+    public int loseBattle() {
+        if (moneyInWallet >= BONUS) {
+            moneyInWallet = moneyInWallet - BONUS;
+        }
```
[the remark this answers is not in the document]
Comment (Dru27) [reply src/main/java/org/fundacionjala/at15/pokemon/Wallet.java]: Good observation!

**Review by eliosfg — APPROVED** [review]
cool!

File: src/main/java/org/fundacionjala/at15/pokemon/Wallet.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    public void winMoney(int money) {
+        moneyInWallet = moneyInWallet + money;
+    }
+
+    public void spendMoney(int money) {
+        if (moneyInWallet >= money) {
+            moneyInWallet = moneyInWallet - money;
+        }
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/Wallet.java:19]: then could be
`else {
    moneyInWallet = 0;
}`

**Pull request merged**